citadel-drone/src/providers/docker.ts

```diff
@@ -127,6 +127,8 @@ class DockerProvider implements BaseProvider {
     });
 
     const user = userInfo();
+    user.uid = user.uid === -1 ? 0 : user.uid;
+    user.gid = user.gid === -1 ? 0 : user.gid;
```

Comment:
Nice, however; I prefer that the default value to be 1000 for the first user of the os. BTW this is a temporary solution since I think this won't work if a user uses WSL with multiple users :) but we will see for the windows full support later.